superset-frontend/src/explore/components/RowCountLabel.tsx

```diff
@@ -31,11 +31,20 @@ const propTypes = {
   loading: PropTypes.bool,
 };
```

Comment:
Now that we use typescript we don't need propTypes anymore 🙂

Reply:
Thanks for the clarification. I am not fully aware of project standards whether `propTypes` is not used in an unusual way, and [have found TypeScript components in Superset](https://github.com/apache/superset/blob/master/superset-frontend/src/explore/components/controls/VizTypeControl/index.tsx) to retain `propTypes` sometime.

Reply:
Well spotted! I'm quite sure that's a mistake